### PR TITLE
Add grafana icon to links, refactor svgs, link to ResourceDetail component

### PR DESCRIPTION
**Pull request by rmars**
This PR started out as a PR to link to our Resource Detail dashboard in addition to grafana in the resource list pages, but I decided to refactor the way we deal with our svgs since I was here.

This branch:
- modifies the GrafanaLink component to consist of the grafana icon that links to grafana
- adds links to the ResourceDetail page in all our metrics tables
- adds a jsx component we can use to wrap svgs so that we don't get annoying 404s on images that we have to handle
- remove the relative paths hack for images
- removes unused svg files in /img

Follow up to #1476

screen shot 2018-08-16 at 5 38 17 pm

Prior art: https://github.com/linkerd/linkerd2/pull/678

This gets rid of the annoying js errors on first pageload:
screen shot 2018-08-17 at 2 33 32 pm

### CODE REVIEW

File: web/app/js/components/MetricsTable.jsx [hunk 1]
```diff
@@ -83,11 +83,16 @@ const columnDefinitions = (resource, namespaces, onFilterClick, showNamespaceCol
...
-                PrefixedLink={PrefixedLink} />
+              <React.Fragment>
+                <PrefixedLink to={"/namespaces/" + row.namespace + "/" + resource + "s/" + row.name}>
+                  {row.name}
+                </PrefixedLink>
+                <GrafanaLink
+                  name={row.name}
+                  namespace={row.namespace}
+                  resource={resource}
+                  PrefixedLink={PrefixedLink} />
```
Comment (klingerf) [inline web/app/js/components/MetricsTable.jsx]: It feels a little bit too subtle to me that the text links to one page and the icon links to a different site altogether, especially when we display them in line with each other:

image

I think it would be cool to put all of the grafana icons in their own column. Maybe we could add a "DASH" column as the last column in each table, and add the icon there for resources that have a corresponding dashboard?

File: web/app/package.json [hunk 2]
```diff
@@ -30,8 +30,8 @@
     "sinon": "^4.1.2",
     "sinon-chai": "^2.14.0",
     "sinon-stub-promise": "^4.0.0",
-    "webpack": "^4.1.12",
-    "webpack-dev-server": "^3.1.4",
+    "webpack": "^4.16.5",
```
Comment (dadjeibaah) [inline web/app/package.json]: More of a nit, I would've preferred for this to be in separate PR, but the change is so small enough that I think its fine.

File: web/app/js/components/MetricsTable.jsx (hunk 1 above)
Comment (rmars) [reply web/app/js/components/MetricsTable.jsx]: sure! that's fine by me, and  I think that's how we had it back in the day when we originally introduced the grafana links!

Comment (rmars) [reply web/app/js/components/MetricsTable.jsx]: screen shot 2018-08-17 at 1 56 26 pm
